app/products/calls/actions/calls.ts

```diff
@@ -265,6 +267,30 @@ export const joinCall = async (
 
     try {
         await connection.waitForPeerConnection();
+
+        // Follow the thread.
+        const database = DatabaseManager.serverDatabases[serverUrl]?.database;
+        if (!database) {
+            return {data: channelId};
+        }
+
+        const isCRTEnabled = await getIsCRTEnabled(database);
+        if (!isCRTEnabled) {
+            return {data: channelId};
+        }
```

Comment:
Is this a client check (e.g. display preferences) or actually checking if the feature is available? I think the goal here was to follow regardless because if at any point the user decides to use CRT they won't have the thread followed. That's what we are doing on the web side anyway so we'd like to be consistent if possible.

Reply:
You should not follow a thread if the user has crt disabled

Reply:
I don't think that's how the feature was designed though. The preference setting is a purely client-side display detail. What really counts is the value of `ServiceSettings.ThreadAutoFollow` on the backend, which defaults to true. And that means that if a user replies to a post they'll be automatically following it regardless of their preference and that's done exactly to avoid the problem I mentioned above.

Reply:
Then I guess is fine to remove the check. cc: @cpoile